### PR TITLE
Development ver2.5

### PR DESCRIPTION
**Pull request by pbogadi**
Try to Accept

### CODE REVIEW

File: 3rdfile.txt [hunk 1]
```diff
@@ -1 +1 @@
-this is the third file that I had added into the project
```
Comment (ksrbigdata) [inline 3rdfile.txt]: what the hell is this????????????????

File: 1stfile.txt [hunk 2]
```diff
@@ -1 +1,2 @@
-this is the first file t
\ No newline at end of file
+this is the first file t.
```
Comment (ksrbigdata) [inline 1stfile.txt:4]: what the hell is this--22222222

File: 3rdfile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+this is the third file
```
Comment (ksrbigdata) [inline 3rdfile.txt]: 1st comment from ksrbigdata

Comment (ksrbigdata) [inline 3rdfile.txt]: this is a line note

Comment (preethambogadi) [inline 3rdfile.txt]: but observe this carefully.. from preetham

**Pull request merged**